packages/lb-annotation/src/core/toolOperation/rect2DOperation.ts

```diff
@@ -0,0 +1,22 @@
+import RectOperation from './rectOperation';
+
+class Rect2DOperation extends RectOperation {
```

Comment:
Please rename the classname. Rect2DOperation is same as RectOperation. And write more comment about this